app/src/main/java/org/wikipedia/search/RecentSearchDatabaseTable.java

```diff
@@ -1,62 +0,0 @@
-package org.wikipedia.search;
-
-import android.content.ContentValues;
-import android.database.Cursor;
-
-import androidx.annotation.NonNull;
-
-import org.wikipedia.database.DatabaseTable;
-import org.wikipedia.database.column.Column;
-import org.wikipedia.database.contract.SearchHistoryContract;
-import org.wikipedia.database.contract.SearchHistoryContract.Col;
-
-import java.util.Date;
-
-public class RecentSearchDatabaseTable extends DatabaseTable<RecentSearch> {
-    private static final int DB_VER_INTRODUCED = 5;
```

Comment:
@sharvaniharan  This constant was `5`, but you changed it to `0` in the converted code. Why?

Reply:
@dbrant did not change this... just did not verify after the auto convert 😱

Reply:
I went back to the code before this change and tried the auto-conversion to Kotlin.  It does not change the 5 to a 0.